dis_snek/models/discord_objects/invite.py

```diff
@@ -66,6 +66,9 @@ def _process_dict(cls, data: Dict[str, Any], client: "Snake") -> Dict[str, Any]:
         if "target_application" in data:
             data["target_application"] = Application.from_dict(data, client)
 
+        if "target_event_id" in data:
+            data["scheduled_event"] = data["target_event_id"]
+
         if channel := data.pop("channel", None):
```

Comment:
Why are you checking for an assignment expression when it will already provide `None` for the actual channel data?

Reply:
I am confused about what you are saying.

Reply:
Oh wait, you mean I should do something like this?
```py
        if data["target_event_id"]:
            data["scheduled_event"] = data["target_event_id"]
```